lms/static/js/edxnotes/views/shim.js

```diff
@@ -58,6 +58,31 @@ define([
         return (timeToExpiry > 0) ? timeToExpiry : 0;
     };
 
+    Annotator.Plugin.Tags.prototype.pluginInit = _.compose(
+        function() {
+            // Add screen reader label for edit mode.
+            var index = this.annotator.editor.fields.length -1;
+            var tagsField = 'annotator-field-' + index;
+            $('<label class="sr" for='+ tagsField +'>' + _t('Edit tags') + '</label>').insertBefore(
```

Comment:
It ends up looking OK. This is what it looks like (in the change I'm pushing up soon to address the issue that Mark found):

<label class="sr" for="annotator-field-1">Tags (space-separated)</label>